dspace-api/src/main/java/org/dspace/authorize/AuthorizeServiceImpl.java

```diff
@@ -309,7 +309,7 @@ protected boolean authorize(Context c, DSpaceObject o, int action, EPerson e, bo
                 }
 
                 if ((rp.getGroup() != null)
-                        && (groupService.isMember(c, rp.getGroup())))
+                        && (groupService.isMember(c, e, rp.getGroup())))
```

Comment:
We would also need to add the `userToCheck` variable to the `isAdmin(c, adminObject)` call on line 257